docs/source/tutorials/datalab/workflows.ipynb

```diff
@@ -670,7 +683,7 @@
     "\n",
     "After detecting non-iid issues, you might be interested in quantifying the likelihood that your dataset is non-iid.\n",
     "\n",
-    "To check if your data is non-iid, Datalab computes a p-value. A low p-value (close to 0) indicates strong evidence against the null hypothesis that the data is iid, suggesting significant dependencies or variations in\n",
+    "To check if your data is non-iid, `Datalab` computes a p-value. A low p-value (close to 0) indicates strong evidence against the null hypothesis that the data is iid, suggesting significant dependencies or variations in\n",
     "distribution across the dataset."
```

Comment:
```suggestion
    "To check if your data is non-iid, `Datalab` computes a p-value. A low p-value (close to 0) indicates strong evidence against the null hypothesis that the data is iid, either because the data appear to be drifting in distribution or inter-dependent across samples."
```